posthog/hogql_queries/test/test_error_tracking_query_runner.py

```diff
@@ -142,6 +142,37 @@ def test_column_names(self):
             ],
         )
 
+    @snapshot_clickhouse_queries
+    def test_search_query(self):
+        _create_event(
+            distinct_id=self.distinct_id_one,
+            event="$exception",
+            team=self.team,
+            properties={
+                "$exception_fingerprint": ["DatabaseNotFound"],
+                "$exception_type": "DatabaseNotFound",
+                "$exception_message": "this is the same error message",
+            },
+        )
+        flush_persons_and_events()
+
+        runner = ErrorTrackingQueryRunner(
+            team=self.team,
+            query=ErrorTrackingQuery(
+                kind="ErrorTrackingQuery",
+                fingerprint=None,
+                dateRange=DateRange(),
+                filterTestAccounts=True,
+                searchQuery="databasenot",
+            ),
+        )
+
+        results = self._calculate(runner)["results"]
+
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0]["fingerprint"], ["DatabaseNotFound"])
+        self.assertEqual(results[0]["occurrences"], 1)
```

Comment:
I think the negative case is only implicitly tested e.g. what happens when the search query doesn't match - do we get 0 results?